net-misc/wsdd/wsdd-0.5.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_REQ_USE="xml(+)"
+
+inherit python-r1 systemd
+
+DESCRIPTION="A Web Service Discovery host daemon."
+HOMEPAGE="https://github.com/christgau/wsdd"
+SRC_URI="https://github.com/christgau/wsdd/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="samba"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}"
```

Comment:
PYTHON_DEPS is really a BDEPEND isn't it? I don't think the targets python interpreter matters until runtime.

Reply:
Mh. Not sure if I get you right, but https://dev.gentoo.org/~mgorny/python-guide/multi.html shows the following: 
```sh
RDEPEND="${PYTHON_DEPS}"
DEPEND="${RDEPEND}"
# and later
BDEPEND="${PYTHON_DEPS}"
```
so PYTHON_DEPS is - from my understanding - not only related to BDEPEND. And since the python interpreter is not required during "build" 'cause there is no actual build step (which you point out), I explicitly set BDEPEND to an empty string (see below). Or do you mean that DEPEND could be left empty/unset for the very same reason? Consequently we will have:

```sh
DEPEND=""
BDEPEND=""
RDEPEND="${PYTHON_DEPS} samba? ( net-fs/samba )"

Reply:
I think you are right, the DEPEND and BDEPEND should be empty.

@mgorny does that sound correct?